src/style.css

```diff
@@ -0,0 +1,29 @@
+.body__page {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star-active.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--0 > .stars__star:nth-child(n + 1),
+.stars--1 > .stars__star:nth-child(n + 2),
+.stars--2 > .stars__star:nth-child(n + 3),
+.stars--3 > .stars__star:nth-child(n + 4),
+.stars--4 > .stars__star:nth-child(n + 5),
+.stars--5 > .stars__star:nth-child(6 - n) {
+  background-image: url(./images/star.svg);
```

Comment:
Modifiers don't work as expected. Make sure that each block has the correct amount of active stars.
Your demo:
![image](https://github.com/mate-academy/layout_stars/assets/96409563/09aec2c9-03df-486c-a0b1-f3b7a86b578c)
Expected result:
![image](https://github.com/mate-academy/layout_stars/assets/96409563/dad70c9e-4c56-4f67-86b2-e151607aef5c)